src/components/Tips.tsx

```diff
@@ -0,0 +1,52 @@
+import { useState } from 'react';
+
+import BulbSvg from '../../public/static/svg/bulb';
+
+type TipCategory = 'general' | 'calculator' | 'decoder';
+
+const tips: { [key in TipCategory]: string[] } = {
```

Comment:
I'd argue this dict is redundant.

Imagine we have a thousand pages.
We'd have to keep all tips here in a huge dictionary, which gets bundled with every other page.
To update a tip, I have to open a page, copy it's ID and then go to this file to look up the ID.

Wouldn't it be easier to keep the calculator tips in the calculator page and the decoder tips in the decoder page?

---

A good rule of thumb for an answer to question "Should I keep it in XML or should I keep it in JSON?" is "Are you building an user interface, or is it serialized data to store or send over the wire?"